src/spikeinterface/preprocessing/filter_gaussian.py

```diff
@@ -32,19 +34,23 @@ class GaussianBandpassFilterRecording(BasePreprocessor):
 
     name = "gaussian_bandpass_filter"
 
-    def __init__(self, recording: BaseRecording, freq_min: float = 300.0, freq_max: float = 5000.0):
+    def __init__(
+        self, recording: BaseRecording, freq_min: float = 300.0, freq_max: float = 5000.0, margin_sd: float = 5.0
```

Comment:
```suggestion
        self, recording: BaseRecording, freq_min: float = 300.0, freq_max: float = 5000.0, margin_sd: float = 6.0
```
?

Reply:
I think 6 by default is too high. Even 5 might be too high and 4 should be completely fine (there's nothing after 4 sd in a Gaussian distribution).

This should help speed  things up a bit.

Reply:
For reference, at 30kHz, a gaussian filter at 300 Hz would take `5*15.92 = 80` ms margins (compared to the 5ms of the "regular filter")

Reply:
Just wanted to point out that there was a mismatch between the default in the init and in the doc! ;)